src/Microsoft.Health.SqlServer/Features/Schema/SchemaVersionStatus.cs

```diff
@@ -11,8 +11,16 @@ namespace Microsoft.Health.SqlServer.Features.Schema
     [JsonConverter(typeof(StringEnumConverter))]
     public enum SchemaVersionStatus
     {
-        Started,
-        Completed,
-        Failed,
+        // Note - Updating SchemaVersionStatus to lower case to be consistent with the statuses applied when schema AutomaticUpdates is set to true
+
+#pragma warning disable SA1300 // Element should begin with upper-case letter
+        started,
+#pragma warning restore SA1300 // Element should begin with upper-case letter
```

Comment:
I am wondering, if we could have warning disable at the top and warning restore at the bottom and avoid the intermediate ones?

Reply:
just updated